### ISSUE TITLE
Table with tomograms

### ISSUE THREAD

**Issue by tischi** [mobie/covid-tomo-datasets#1]
@constantinpape 
If you think the label mask is too much of a hack, we can also think of another solution.
But maybe it actually turns out to even be a good general solution even for the PlateViewer...
I can't wrap my head around it right now. Maybe we just try and see what happens?

**Comment by constantinpape** [comment]
> If you think the label mask is too much of a hack, we can also think of another solution.
> But maybe it actually turns out to even be a good general solution even for the PlateViewer...
> I can't wrap my head around it right now. Maybe we just try and see what happens?

I agree, let's just try it and see how we can improve it in the long run.
I think for whatever we come up with we can reuse the table in any case, because we will somehow need the bounding box and anchor information etc.

**Comment by constantinpape** [comment]
I have added the mock segmentation with tables for 3 of the datasets.
(There is some issue with the remaining one, but this should be easy to figure out with Yannick).

**Comment by tischi** [comment]
@constantinpape 

<img width="800" alt="image" src="https://user-images.githubusercontent.com/2157566/150737035-32884821-4994-4926-885f-2e6ea9918520.png">

I think the name of the uiGroup and "segmentation" is not very intuitive:

“position: em-tomogram-segmentation” as this is just the annotation of the images.

BTW: Could it be that the "new spec" branch still needs to be merged into main?

**Comment by constantinpape** [comment]
> I think the name of the uiGroup and "segmentation" is not very intuitive:

Sure, that's easy to fix; I can do that later.

**Comment by constantinpape** [comment]
> BTW: Could it be that the "new spec" branch still needs to be merged into main?

I am not sure about the differences between the two branches, will check it out later.

**Comment by tischi** [comment]
If you can, could you do it before lunch? I would like to demo this in a COMULIS meeting today...

**Comment by constantinpape** [comment]
Sorry, I am still traveling to Heidelberg and the Bahn is misbehaving...
So I don't have so stable internet and can probably not do this now.

**Comment by tischi** [comment]
I think before 15:00 would still be fine.
Otherwise I can live with the current version.

**Comment by constantinpape** [comment]
I had a quick look but the differences between the `main` branch and `new-spec` are a bit complicated and I am not quite sure which one is more up-to-date. I need to double check this while I can open the project in MoBIE, which is not possible with train WiFi... 
Let's stick with the current version and I will have a look at this once I work on the corresponding figure for the paper.

**Comment by tischi** [comment]
OK!
spec-v2 seems more up to date for me.
I think master is still using a hack for the annotated image view.
